plugin/queries.js

```diff
@@ -42,10 +42,30 @@ module.exports.CREATE_OPERATION = `
               id
               title
               handle
+              description
+              productType
+              media {
```

Comment:
Do we need to download the image [the way that we do in the existing plugin](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-shopify/src/nodes.js#L69-L85)?

Reply:
@sslotsky Ah! I misunderstood and thought that work was to be done in a separate PR. I'll get that addressed in this one :+1:

Reply:
Well for me it depends on whether or not the swag store will work without doing that. If it works, we can do that in a different PR. But if we need it for the swag store, then let's get it in this one.